server/routes/index.js

```diff
@@ -1290,4 +1292,89 @@ router.route('/charts/new')
       return res.status(500).send({ message: err.message })
     }
 })
+
+/**
+ * Study Details
+ */
+router.route('/study-details')
+  .get(ensureAuthenticated, async (req, res) => {
+    try {
+      const { display_name, role, icon } = req.session;
+      console.log(req.user)
+      return res.status(200).send(studyDetailsPage({
+        uid: req.user,
+        name: display_name,
+        role,
+        icon
+      }))
+    } catch (error) {
+      console.error(error.message)
+
+      return res.status(500).send({ message: err.message })
+    }
+  })
+
+router.route('/api/v1/study-details')
+  .get(ensureAuthenticated, async(req, res) => {
+    try {
+      const data = await mongoData
+        .collection(collections.studyDetails)
+        .find({ owner: req.user })
+        .toArray()
+
+      return res.status(200).json({ data })
+    } catch (error) {
+
+      return res.status(500).json({ message: error.message })
+    }
+  })
+
+  router.route('/api/v1/study-details/:detailId')
+    .delete(ensureAuthenticated, async(req, res) =>{
+      const { detailId } = req.params;
+      try {
+        const deleted = await mongoData
+          .collection(collections.studyDetails)
+          .deleteOne({ _id: ObjectID(detailId) })
+
+      if (deleted.deletedCount > 0) {
+
+        return res.status(200).json({ data: deleted.deletedCount });
+      } else {
+
+        return res.status(404).json({ message: 'Study Details Not Found' });
+      }
+    } catch (error) {
+
+      return res.status(500).json({ message: error.message })
+    }
+  })
+
+router.route('/api/v1/study-details/upload')
```

Comment:
Since we're using a different HTTP method (`post`) we can keep the path the same. This pattern is similar to restful resources in rails but is a nice convention as it allows us to take CRUD actions on a resource that is defined in the route (study details in this case).

```suggestion
router.route('/api/v1/study-details')
```

Reply:
[f682564](https://github.com/AMP-SCZ/dpdash/pull/81/commits/f682564832e4de21cbe2ec9f3f4245bfeaef011a)